mlflow/openai/utils.py

```diff
@@ -151,6 +151,13 @@ def _validate_model_params(task, model, params):
         )
 
 
+def _exclude_params_from_envs(params, envs):
```

Comment:
Can we add a parametrized test for the behavior of this override to ensure that param entries override env variable entries as expected (checking for things like None values in param overrides so that we have effective error handling for situations like that)